service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/DraftOrdersApprovedEventHandlerTest.java

```diff
@@ -73,6 +73,7 @@
 
 @ExtendWith(SpringExtension.class)
```

Comment:
vv minor, looks like this can just be mockito extension instead of spring extension